app/views/proprietor/accounts/edit.html.erb

```diff
@@ -51,9 +51,9 @@
           <% end %>
 
           <% unless @account.search_only? %>
-            <h3><%= t(".datacite_endpoint") %></h3>
-					  <%= f.fields_for :datacite_endpoint do |s| %>
-						  <%= s.input :mode, required: false  %>
+            <h3><%= t(".data_cite_endpoint") %></h3>
+					  <%= f.simple_fields_for :data_cite_endpoint do |s| %>
+						  <%= s.input :mode, collection: ['test', 'production'], required: false  %>
```

Comment:
Undecided about select input. Will need reviewer to determine if it should be implemented or omitted.